observablehq.config.ts

```diff
@@ -107,7 +107,7 @@ export default {
   }
 <script type="module">/Win/.test(navigator.platform) || Array.from(document.querySelectorAll(".win"), (e) => e.remove())</script>`,
   header: `<div style="display: flex; align-items: center; gap: 0.5rem; height: 2.2rem; margin: -1.5rem -2rem 2rem -2rem; padding: 0.5rem 2rem; border-bottom: solid 1px var(--theme-foreground-faintest); font: 500 16px var(--sans-serif);">
-  <a href="https://observablehq.com/" style="display: flex; align-items: center;">
+  <a href="https://observablehq.com/" target="self" style="display: flex; align-items: center;">
```

Comment:
MDN mentions `_self`
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener

Reply:
Ah, oops, yes, this should be `_self`.